src/javascripts/domConstruct.js

```diff
@@ -0,0 +1,108 @@
+const oCommentUtilities = require('o-comment-utilities');
+const generateId = oCommentUtilities.generateId;
+
+function getCamelCaseName (str) {
+	const parts = str.split('-');
+	let result = parts[0];
+
+	for (let i = 1; i < parts.length; i++) {
+		result += parts[i].charAt(0).toUpperCase() + parts[i].slice(1);
+	}
+
+	return result;
+}
+
+function getAttributeValue (valueRead) {
+	if (valueRead === "true") {
+		return true;
+	} else if (valueRead === "false") {
+		return false;
+	} else {
+		const intValue = parseInt(valueRead, 10);
+		if (!isNaN(valueRead) && intValue) {
+			return intValue;
+		} else {
+			return valueRead;
+		}
+	}
+}
+
+module.exports = function (config) {
+	let el = config.context;
+
+	if (!el) {
+		el = document.body;
+	} else if (!(el instanceof HTMLElement)) {
+		el = document.querySelector(el);
+	}
+
+	const instances = Array.from(el.querySelectorAll('[data-o-component="'+ config.classNamespace +'"]'));
+	const commentInstances = instances.filter(instance => !instance.hasAttribute('data-o-comments-count'));
```

Comment:
FYI, this file was copy/pasted from `o-comment-utilities` and I added this line of code to it. It filters out any instances that have the attribute `data-o-comments-count`.